config/crd/bases/toolchain.dev.openshift.com_usersignups.yaml

```diff
@@ -131,6 +131,11 @@ spec:
                 description: The cluster in which the user is provisioned in If not
                   set then the target cluster will be picked automatically
                 type: string
+<<<<<<< HEAD
```

Comment:
This needs some cleanup after resolving a conflict :)

Reply:
Fixed